lib/count_between.js

```diff
@@ -0,0 +1,11 @@
+function countNumbersBetween(numbers,lowerBound, upperBound){
+  var counter = 0;
+  numbers.forEach(function(number){
+    if(lowerBound < number && number < upperBound){
```

Comment:
you could extract this out into the line above to make it easier to read - or even into individual functions? i think individual functions may be a tad too much, but i like the idea of trying to do this in a better way!